schema/chat_room.proto

```diff
@@ -0,0 +1,343 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+import "google/longrunning/operations.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "google/rpc/error_details.proto";
+
+package google.showcase.v1alpha3;
+
+option go_package = "github.com/googleapis/gapic-showcase/server/genproto";
+option java_package = "com.google.showcase.v1alpha3";
+option java_multiple_files = true;
+
+option (google.api.metadata) = {
+  product_uri: "https://github.com/googleapis/gapic-showcase"
+};
+
+service ChatRoom {
+  rpc CreateRoom(CreateRoomRequest) returns (Room) {
```

Comment:
It will much more resemble real APIs (such as Pub/Sub, Spanner) if the room creation returned `google.longrunning.Operation` and was asyncronous. After all, it's logical since room creation might call some long running operation under the hood.

Reply:
(Luke just pointed out that Pub/Sub RPCs don't return LRO, but still, for me, room creation can be LRO)

Reply:
I think this conversation boils down to how https://github.com/googleapis/gapic-showcase/pull/39#issuecomment-432024845 will be resolved.

Reply:
I would say ~90%-95% of our create methods return the resource (and that is [our expected starting point](https://cloud.google.com/apis/design/standard_methods#create)). We do sometimes have creates that return LROs, so I am happy to have one if it is appropriate.

Given that "create room" is likely to be the first thing some people do, I expect this is the wrong place to introduce an LRO.